modules/lvgl/include/lvgl_zephyr.h

```diff
@@ -32,6 +32,20 @@ extern "C" {
  */
 int lvgl_init(void);
 
+#ifdef CONFIG_LV_Z_RUN_LVGL_ON_WORKQUEUE
+/**
+ * @brief Schedule an LVGL delayed by amount of miliseconds
+ *
+ * This function submit a delayed job to the LVGL internal's
+ * workqueue, the job will be delayed by an amount of miliseconds
+ * selected by the user
+ *
+ * @return 0 on success, negative errno code on failure
+ */
+int lvgl_schedule_job(int32_t ms);
```

Comment:
This is not what I meant, but something like this:

```c
struct k_work_q *lvgl_workqueue(void);
```

This would allow users to have their own `k_work` items submitted/scheduled on the LVGL thread, making it thread-safe to call `lv_` functions there.